app.py

```diff
@@ -24,6 +24,15 @@ def init_app(self):
         """
         Called as the application is being initialized
         """
+        
+        if self.engine.name == "tk-houdini":
```

Comment:
Rather than doing this, it would be better I think to do something like
```
if not self.engine.has_ui:
   return
```
A bit like it's done [here](https://github.com/shotgunsoftware/tk-multi-workfiles2/blob/a65f8d4da2e2c6d6aa98f663e83295e95d234902/app.py#L92), that makes it engine agnostic. 
Also that line I just linked could then lose the UI check, as you would be breaking out sooner if didn't have a UI.

Reply:
Phil's correct here. With that done I feel like this is good to go. Like Phil thought, this seems like the right place to put this logic.